Controllers/twitterController.js

```diff
@@ -0,0 +1,11 @@
+const socialMedia = {};
```

Comment:
You're still using capital letters for your folder names...

Reply:
This has been changed

Reply:
Did you push the changes? Still showing a capital C...

Reply:
I have, just pushed again on command line and response was everything was up to date